impl/maven-impl/src/test/java/org/apache/maven/impl/DefaultPluginXmlFactoryTest.java

```diff
@@ -73,9 +74,9 @@ void readFromInputStreamParsesPluginDescriptorCorrectly() {
         PluginDescriptor descriptor = defaultPluginXmlFactory.read(XmlReaderRequest.builder()
                 .inputStream(new ByteArrayInputStream(SAMPLE_PLUGIN_XML.getBytes()))
                 .build());
-        assertThat(descriptor.getName()).isEqualTo(NAME);
-        assertThat(descriptor.getGroupId()).isEqualTo("org.example");
         assertThat(descriptor.getArtifactId()).isEqualTo("sample-plugin");
+        assertThat(descriptor.getGroupId()).isEqualTo("org.example");
```

Comment:
I don't know why you changed this to reorder it. It looks like needless churn.

Reply:
yes it takes away, please excuse this, need SOC. revert.